.github/workflows/test.yml

```diff
@@ -11,17 +11,19 @@ jobs:
       matrix:
         platform: [ubuntu-latest]
     runs-on: ${{ matrix.platform }}
+    env:
+      WALLET_PRIVATE_KEY: "0x3d3cbc973389cb26f657686445bcc75662b415b656078503592ac8c1abb8810e"
```

Comment:
I think we can remove this if we remove the usage of `accounts: [process.env.WALLET_PRIVATE_KEY || ""],` in `hardhat.config.ts` or not have `|| ""`

The tests do not use it at all so seems out of place here.

Reply:
thx. added a test wallet in the config file to load it by default if env variable is not provided instead of the empty `""`